run.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
What is the usage of this run.sh?

Reply:
I think its [shebang](https://linuxize.com/post/bash-shebang/) to tell which interpreter to parse the script if running script as an executable?

Reply:
Err, I meant the entire `run.sh` file itself. Is there some usage of `run.sh` to test the agent either presently or in the future?

Reply:
Oh sorry haha :D opps read it wrong...

Reply:
Right now there isn't much use to it because of changes to the discovery plugin API and smoketest setup. I can remove it for now since the manual testing steps I outlined above aren't so onerous.